### PR TITLE
Digital Root

### PR DESCRIPTION
**Pull request by SriRaghavi11**

### CODE REVIEW

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the number:");
+         int number = int.Parse (Console.ReadLine ());
+         int DigitalRoot = CalculateDigitalRoot (number);
+         Console.WriteLine ($"The digital root of {number} is {DigitalRoot}");
+      }
+      static int CalculateDigitalRoot (int num) {
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]: This code works but there is a simpler way to calculate digital root of a number in base 10 given in the wikipedia article

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs]: Will check and change

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         int digitalroot = n % 9;
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]: image

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         if (n != 0)
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]: What if input is 0?

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 3 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs]: will give with a if else statement and update

File: ConsoleApp1/Program.cs [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
-    }
-}
\ No newline at end of file
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         if (n != 0) {
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]: Digital root is defined only for positive numbers

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 4 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs]: I will change with n>0 and update

File: ConsoleApp1/Program.cs [hunk 5]
```diff
@@ -1,7 +1,17 @@
...
+﻿using System;
+namespace DigitalRootFinder {
+   class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter the number:");
+         int n = int.Parse (Console.ReadLine ());
+         if (n > 0) {
+            int digitalroot = 1 + ((n - 1) % 9);
+            Console.WriteLine ($"The digital root is {digitalroot}");
+         } else if (n==0)
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]:
```suggestion
         } else if (n == 0)
```

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**